modules/ROOT/pages/create-community.adoc

```diff
@@ -103,12 +103,16 @@ To verify that Salesforce CRM Content is enabled:
 
 After you create a community, you must create different user profiles for your community _members_ (signed in) and _guests_, so that you can control access to and visibility of APIs and content.
 
-Before configuring member and guest profiles and permissions, open *Setup* > *User Management Settings* and ensure that *Enhanced Profile User Interface* is enabled.
+Configure the settings with one of these options:
+
+. Configure the member and guest user settings automatically with default settings with the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
+. Configure the member and guest user settings manually with custom settings with these steps.
```

Comment:
These should be bullets, not steps, because they are not chronological.

Reply:
```suggestion
* Configure the member and guest user settings automatically using the default settings specified by the *Fix* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
* Configure the member and guest user settings manually using custom settings applied in a series of steps.
```

Reply:
Thank you! I should have realized that about the bullets. Implemented all changes, including bullets and text changes.